packages/react-components/src/queries/useTimeSeriesData/useTimeSeriesData.ts

```diff
@@ -18,29 +16,18 @@ export type UseTimeSeriesDataOptions = {
   settings?: RequestSettings;
 };
 
-const filterRequests = (requests: Request[]) => {
-  return {
-    assetPropertyValueHistoryRequests: requests,
-  };
-};
-
 export const useTimeSeriesData = ({
```

Comment:
Missing tests for this? It's not going to be easy to write unit tests for a system this large after everything is functionally complete.